### PR TITLE
Feature/user check claims and outstanding balances

### PR DESCRIPTION
**Pull request by gearsix**
I've put the functionality into MembershipController, if it should be somewhere else it should be easy to move. We should probably start consolidating some of the controllers.
- Retrieved member's info, currently using temporary user id.
- Retrieved users Claims, again using temporary user id.
- Changed Claims 'amount' to a double
- Created temporary .jsp for members, currently just lists outstanding balances & active claims
- small fixes in web.xml, had to change membership endpoint to /member - gives 404 otherwise.
This push is a bit messy, so feel free to critique.

### CODE REVIEW

File: XYZDrivers/src/main/java/com/xyzdrivers/repositories/ClaimsRepo.java [hunk 1]
```diff
@@ -61,8 +61,28 @@ public void delete(Claim model) {
...
+            {
+                Claim claim = new Claim(claimData[1].toString(),                    //mem_id
+                                        LocalDate.parse(claimData[2].toString()),   //date
+                                        claimData[3].toString(),                    //rational
+                                        claimData[4].toString(),                    //status
+                                        (double)claimData[5]);                       //amount
+                claims.add(claim);
+            }
+        } catch (SQLException | IllegalArgumentException ex) {
+            //@TODO
```
Comment (bkneis) [inline XYZDrivers/src/main/java/com/xyzdrivers/repositories/ClaimsRepo.java]: Please add Logger....

File: XYZDrivers/src/main/java/com/xyzdrivers/models/Claim.java [hunk 2]
```diff
@@ -42,7 +42,7 @@ public String getStatus() {
         return this.status;
     }
 
-    public float getAmount() {
+    public double getAmount() {
```
Comment (bkneis) [inline XYZDrivers/src/main/java/com/xyzdrivers/models/Claim.java:45]: Why was this changed from float to double?

Comment (gearsix) [reply XYZDrivers/src/main/java/com/xyzdrivers/models/Claim.java:45]: It's a DOUBLE in the SQL db, so converting it to a float was throwing an error.

Comment (gearsix) [reply XYZDrivers/src/main/java/com/xyzdrivers/models/Claim.java:45]: retrieving it as a float was throwing an error*, not converting.
Regardless, I figured it would be better to change the type than convert since it's a double in the DB.

Comment (bkneis) [reply XYZDrivers/src/main/java/com/xyzdrivers/models/Claim.java:45]: Ah right, I thought it was a float for some reason. Yeah change to double was good shout

**Review by bkneis — APPROVED** [review]
LGTM
